### PR TITLE
Meeting info to README.md

### PR DESCRIPTION
**Pull request by surajssd**
added link to agenda doc and meeting bluejeans link

### CODE REVIEW

**Comment by surajssd** [comment]
@sebgoa done! I missed that part, sorry about that :)

File: README.md [hunk 1]
```diff
@@ -97,6 +97,10 @@ You can reach the maintainers of this project at:
 
 Incubation of `kompose` into the Kubernetes project will be discussed in the [API Machinery SIG](https://github.com/kubernetes/community)
 
+We do a bi-weekly community meeting. Here is the link to [agenda doc](https://docs.google.com/document/d/1I5I21Cp_JZ9Az5MgMcu6Hl7m8WQ1Eqk_WeQLHenNom0/edit?usp=sharing).
+
+Meeting link: https://bluejeans.com/404059616
```
Comment (sebgoa) [inline README.md]: did not figure out how to update a review....but the bluejeans stuff should be a link ;)

**Comment by sebgoa** [comment]
LGTM, @surajssd go ahead and merge

**Comment by surajssd** [comment]
@sebgoa you will have to resolve it, right now merging is blocked!

**Comment by surajssd** [comment]
Oh now it shows resolved, merging

**Pull request merged**